src/igvfd/schemas/changelogs/biosample.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog for *`biosample.json`*
 
```

Comment:
i thikn we need to update the changelogs of the concrete classes (tissue, etc.) with minor chnages

Reply:
done